cmake/modules/SearchInstalledSoftware.cmake

```diff
@@ -1498,6 +1498,28 @@ else()
   set(tmva-rmva  OFF CACHE BOOL "Disabled because 'tmva' is disabled (${tmva-rmva_description})"  FORCE)
 endif()
 
+#---Check for Pyroot---------------------------------------------------------------------
+if(pyroot)
```

Comment:
@oshadura should we also tackle the pymva case, which is only a few lines up in this file? Pymva also depends on python-dev, @lmoneta can confirm, so we should deactivate it too if python-dev is not found. Right now, it only checks for numpy.

Reply:
Yup! Lets add the same lines there:
`  if(fail-on-missing AND (NOT PYTHONLIBS_FOUND AND NOT Python2_Development_FOUND AND NOT Python3_Development_FOUND))
    message(FATAL_ERROR "PyROOT: Python development package not found and pyroot component required"
                        " (python executable: ${PYTHON_EXECUTABLE})")
  elseif(NOT PYTHONLIBS_FOUND AND NOT Python2_Development_FOUND AND NOT Python3_Development_FOUND)
    message(STATUS "PyROOT: Python development package not found for python ${PYTHON_EXECUTABLE}. Switching off pyroot option")
    set(pyroot OFF CACHE BOOL "Disabled because Python development package was not found" FORCE)
  endif()`